composer.json

```diff
@@ -5,18 +5,18 @@
   "license": "proprietary",
   "require": {
     "php": ">=7.1",
-    "spryker-shop/cart-page": "^1.0.0",
+    "spryker-shop/cart-page": "^1.0.0 || ^3.0.0",
     "spryker-shop/checkout-page": "^1.0.0 || ^2.0.0",
     "spryker-shop/quick-order-page-extension": "^1.1.0",
     "spryker-shop/shop-application": "^1.0.0",
     "spryker-shop/shop-ui": "^1.19.0",
     "spryker/application": "^3.0.0",
-    "spryker/cart": "^4.0.0 || ^5.0.0",
+    "spryker/cart": "^4.0.0 || ^5.0.0 || ^7.0.0",
```

Comment:
Composer update --prefer-lowest

`Updating spryker/cart (7.0.0 => 5.0.0): Loading from cache`

We should remove support for v4 as it will not be installed anyway.

Reply:
I agree. Will you remove it and approve?

Reply:
Wait
If this is in master like this you cant restrict it here IMO.
We should be fixing master instead soon.